db.go

```diff
@@ -534,8 +534,9 @@ func (db *DB) reload() (err error) {
 	sort.Slice(blocks, func(i, j int) bool {
 		return blocks[i].Meta().MinTime < blocks[j].Meta().MinTime
 	})
-	if err := validateBlockSequence(blocks); err != nil {
```

Comment:
I am not sure about changing anything in `db.go` before resolving conflicts (: I would aim for changing as little as possible. (e.g `-type Overlaps map[TimeRange][]BlockMeta` -> `type Overlaps map[TimeRange][]*Block` change)